Formula/trafficserver.rb

```diff
@@ -48,7 +70,7 @@ def install
       --enable-experimental-plugins
     ]
 
-    system "autoreconf", "-fvi" if build.head?
+    system "autoreconf", "-fvi" if build.head? || OS.mac?
```

Comment:
This worked for me locally on Monterey.
Could also just `autoreconf` everywhere including Linux (as only Linux stable isn't running it right now. Not sure if it builds yet).